Assets/MixedRealityToolkit/Inspectors/MixedRealityWireframeShaderGUI.cs

```diff
@@ -0,0 +1,210 @@
+        protected MaterialProperty depthTest;
+        protected MaterialProperty depthWrite;
+        protected MaterialProperty depthOffsetFactor;
+        protected MaterialProperty depthOffsetUnits;
+        protected MaterialProperty colorWriteMask;
+        protected MaterialProperty cullMode;
+        protected MaterialProperty renderQueueOverride;
+
+        protected MaterialProperty baseColor;
+        protected MaterialProperty wireColor;
+        protected MaterialProperty wireThickness;
+
+        public override void OnGUI(MaterialEditor materialEditor, MaterialProperty[] props)
+        {
+            Material material = (Material)materialEditor.target;
+
+            FindProperties(props);
+            Initialize(material);
+
+            RenderingModeOptions(materialEditor);
+
+            GUILayout.Label(Styles.mainPropertiesTitle, EditorStyles.boldLabel);
+            materialEditor.ShaderProperty(baseColor, Styles.baseColor);
+            materialEditor.ShaderProperty(wireColor, Styles.wireColor);
+            materialEditor.ShaderProperty(wireThickness, Styles.wireThickness);
+
+            AdvancedOptions(materialEditor, material);
+
+            //base.OnGUI(materialEditor, props);
+        }
+
+        protected void FindProperties(MaterialProperty[] props)
+        {
+            renderingMode = FindProperty(BaseStyles.renderingModeName, props);
+            customRenderingMode = FindProperty(BaseStyles.customRenderingModeName, props);
+            sourceBlend = FindProperty(BaseStyles.sourceBlendName, props);
+            destinationBlend = FindProperty(BaseStyles.destinationBlendName, props);
+            blendOperation = FindProperty(BaseStyles.blendOperationName, props);
+            depthTest = FindProperty(BaseStyles.depthTestName, props);
+            depthWrite = FindProperty(BaseStyles.depthWriteName, props);
+            depthOffsetFactor = FindProperty(BaseStyles.depthOffsetFactorName, props);
+            depthOffsetUnits = FindProperty(BaseStyles.depthOffsetUnitsName, props);
+            colorWriteMask = FindProperty(BaseStyles.colorWriteMaskName, props);
+
+            cullMode = FindProperty("_CullMode", props);
+            renderQueueOverride = FindProperty("_RenderQueueOverride", props);
+
+            baseColor = FindProperty("_BaseColor", props);
+            wireColor = FindProperty("_WireColor", props);
+            wireThickness = FindProperty("_WireThickness", props);
+        }
+
+        public override void AssignNewShaderToMaterial(Material material, Shader oldShader, Shader newShader)
+        {
+            float? cullMode = GetFloatProperty(material, "_Cull");
+
+            base.AssignNewShaderToMaterial(material, oldShader, newShader);
+   
+            SetFloatProperty(material, null, "_CullMode", cullMode);
+
+            // Setup the rendering mode based on the old shader.
+            if (oldShader == null || !oldShader.name.Contains("Legacy Shaders/"))
```

Comment:
>"Legacy Shaders/ [](start = 62, length = 16)

Nit: Consider moving some of the shared string definitions to the base class, so that if we ever need to change this (however unlikely) we can reduce the amount of copy paste/chance for error then.